src/mmw/js/src/modeling/views.js

```diff
@@ -114,6 +129,12 @@ var ScenarioTabPanelView = Marionette.ItemView.extend({
         role: 'presentation'
     },
 
+    onDomRefresh: function() {
+        if (this.model.get('active')) {
+            this.$el.addClass('active');
+        }
+    },
```

Comment:
I think this is no longer needed because of https://github.com/lliss/model-my-watershed/blob/feature/lol/scenario-dropdown/src/mmw/js/src/modeling/views.js#L138.